accounting/invoice/templates/default/default.html

```diff
@@ -0,0 +1,134 @@
+              <span id="invoice-created">Created: {{ invoice.date|date:"F d, Y" }}</span><br>
+              <span id="invoice-billing-dates">Billed for {{ invoice.billing_start_date|date:"F d, Y" }} to {{ invoice.billing_end_date|date:"F d, Y" }}</span>
+              <span id="invoice-due">Due: {{ invoice.due_date|date:"F d, Y" }}</span>
+            </div>
+          </div>
+          <div class="row">
+            {% for user_str, user in users %}
+              <div class="col-xs-6">
+                <div id="{{ user_str }}-details">
+                  <h5 class="user-header"><strong>{{ user_str|upper }}</strong></h5>
+                  <span class="user-name">{{ user }}</span><br>
+                  <span class="user-address-line-1">{{ user.address.address_line1 }}</span><br>
+                  {% if user.address.address_line2 %}
+                    <span class="user-address-line-2">{{ user.address.address_line2 }}</span><br>
+                  {% endif %}
+                  <span class="user-city">{{ user.address.city }}</span>,
+                  {% if user.address.state %}
+                    <span class="user-state">{{ user.address.state }}</span>,
+                  {% endif %}
+                  <span class="user-zipcode">{{ user.address.zipcode }}</span><br>
+                  <span class="user-country">{{ user.address.country.name }}</span>
+                  {% if user.vat %}
+                    <br><span class="user-vat">VAT number: {{ user.vat }}</span>
+                  {% endif %}
+                </div>
+              </div>
+            {% endfor %}
+          </div>
+          <div class="row">
+            <div id="client-bank-account-details" class="col-xs-6">
+              <h5 class="client-bank-account-header"><strong>BANK ACCOUNT</strong></h5>
+              Bank Name: {{ bank_account.bank.name }}<br>
+              Bank Account Type: {{ bank_account.type|title }}<br>
+              Bank Account Currency: {{ bank_account.currency }}<br>
+              {% for name, value in bank_account_details %}
+                {{ name }}: {{ value }}<br>
+              {% endfor %}
+            </div>
+            {% if invoice.extra_text %}
+            <div id="extra" class="col-xs-6">
+              <h5 class="extra-header"><strong>EXTRA</strong></h5>
+              <p id="extra-text">{{ invoice.extra_text|linebreaks }}</p>
+            </div>
+            {% endif %}
+          </div>
+        </header>
+
+        <main>
+          <table id="line-items" class="table table-striped table-condensed" cellpadding="0" cellspacing="0">
+            <tr>
+              <th scope="col" class="column column-key">Key</th>
+              <th scope="col" class="column column-name">Name</th>
+              <th scope="col" class="column column-quantity">Quantity</th>
+              <th scope="col" class="column column-price">Price</th>
+              <th scope="col" class="column column-total">Total</th>
+            </tr>
```

Comment:
@UmanShahzad Should we wrap this row in a `thead`?

Reply:
I tried doing this, but HTML to PDF doesn't handle this well unfortunately: when the table spans multiple pages, the header appears on each page but by doing so it covers other line items -- for some reason the line items aren't below the header on each page.

Reply:
@UmanShahzad That's weird... Thanks for the details, though.